update-lb4-docs.js

```diff
@@ -0,0 +1,40 @@
+const fs = require('fs-extra');
+const path = require('path');
+
+const srcDocs = path.resolve(__dirname,'node_modules/@loopback/docs/site');
+const destDocs = path.resolve(__dirname, 'pages/en/lb4');
+
+// Remove existing LoopBack 4 documentation content
+try {
+  fs.removeSync(destDocs);
+} catch (err) {
+  console.error('failed to cleanup previous docs folder %s', err.stack);
+  process.exit(1);
+}
+
+
+function copyDocs(src, dest) {
+  try {
+    fs.copySync(src, dest);
+  } catch (err) {
+    console.error('failed to copy latest docs %s', err.stack);
+    process.exit(1);
+  }
+}
+
+// copy the latest docs from @loopback/docs to pages/en/lb4 directory
+copyDocs(srcDocs, destDocs);
+
+const fileToUpdate = path.resolve(destDocs, 'Testing-the-API.md');
```

Comment:
Obsolete code?

Reply:
Nah, it is used down the line (see https://github.com/strongloop/loopback.io/pull/643/commits/1ba3aba52978255543499c0ceaa8f429e2f30427#diff-015a67e879f2083136bcd7796abb4f33R34)